manifests/r/RussellBanks/Komac/Nightly/2.0.0/RussellBanks.Komac.Nightly.installer.yaml

```
@@ -1,25 +1,51 @@
+# Created with Komac v1.11.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
 PackageIdentifier: RussellBanks.Komac.Nightly
 PackageVersion: 2.0.0
 InstallerLocale: en-US
-InstallerType: portable
+InstallerType: inno
+Scope: user
 UpgradeBehavior: install
 Commands:
 - komac
-ReleaseDate: 2023-12-21
+ReleaseDate: 2024-01-05
 Installers:
-- Architecture: x64
+- Architecture: x86
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x64-nightly.exe
-  InstallerSha256: c97a9e38345798d522ac4b86f705dfa7d754eb7a74246f29bf03d60b12b3d67c
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: d79d0c766bad4b87ee029e13ba351565e6b1fe362519a8ba4fae4c4d02ab07bb
+  InstallerSwitches:
+    Custom: /CURRENTUSER
 - Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x64.exe
+  InstallerSha256: d79d0c766bad4b87ee029e13ba351565e6b1fe362519a8ba4fae4c4d02ab07bb
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
   Scope: user
-  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/Komac-windows-x86-nightly.exe
-  InstallerSha256: 497f9728035503ca06e412df5b4185c7e51db41280fea7a209657426af365e9b
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 2aae771eff66b784f0a28db99a11264040e219b3caa64abf19dad5ed7523a9ad
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
+  InstallerSha256: 2aae771eff66b784f0a28db99a11264040e219b3caa64abf19dad5ed7523a9ad

```

```suggestion
  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/russellbanks/Komac/releases/download/nightly/KomacSetup-nightly-x86.exe
  InstallerSha256: D79D0C766BAD4B87EE029E13BA351565E6B1FE362519A8BA4FAE4C4D02AB07BB
```
These are opposites of each other, so it seems like they just got swapped.